src/controllers/event_loop.py

```diff
@@ -14,7 +14,7 @@ def event_loop() -> None:
 
     window = Window()
 
-    player = Player([200, 200])
+    player = Player(1000, window.width/2 * .75)
```

Comment:
pylint is complaining about this line. It's just a stylistics issue. pylint checks to make sure all of us are same styles in code.

I would do this:
`(window.width / 2) * 0.75`